framework/tests/viewpointtest/AnnoOnTypeVariableUse.java

```diff
@@ -0,0 +1,17 @@
+import viewpointtest.quals.*;
```

Comment:
Treatment to type variables?
RDM: do receiver-dependent viewpoint adaptation.
add a new qualifier LUB: to do LUB of the type argument and the class, have transitive immutability
RI (mutable/immutable): no rdm, no lub effects.

Reply:
think about the ri subtitution.

Reply:
RI uses type argument information / mutable ignores type argument.

Reply:
Box < T> {
    @mutable T f;
}
@Immutable
class C {
}
Box < @Immutable C>  a;
a.f --> mutable which contradicts to the definition of C.
well-form rules check / or change the semantics on the type variable use.